resolwe_bio/processes/variant_calling/snpdat.yml

```diff
@@ -82,23 +82,23 @@
       echo "{\"proc.progress\":0.05,\"proc.rc\":$?}"
 
       # indexing
-      {{ proc.slugs_path }}/samtools-0.1.19/samtools faidx "${GENOME_NAME}.fasta"
+      samtools faidx "${GENOME_NAME}.fasta"
       echo "{\"proc.progress\":0.1,\"proc.rc\":$?}"
-      java -jar "{{ proc.slugs_path }}/picard-tools-1.118/CreateSequenceDictionary.jar" R="${GENOME_NAME}.fasta" O="${GENOME_NAME}.dict"
+      java -jar "CreateSequenceDictionary.jar" R="${GENOME_NAME}.fasta" O="${GENOME_NAME}.dict"
       echo "{\"proc.progress\":0.15,\"proc.rc\":$?}"
 
       # filtering
-      echo "su $(whoami) -s /bin/bash -c \"java -jar {{ proc.slugs_path }}/GATK-3.2.2/GenomeAnalysisTK.jar -T VariantFiltration -V '{{variants.vcf.file}}' -o '${VARIANTS}_filtered.vcf' -R '${GENOME_NAME}.fasta' --filterName DepthFilter --filterExpression 'DP < {{filtration.rd}} || AF < {{filtration.af}}'\"" > temp.sh
+      echo "su $(whoami) -s /bin/bash -c \"java -jar GenomeAnalysisTK.jar -T VariantFiltration -V '{{variants.vcf.file}}' -o '${VARIANTS}_filtered.vcf' -R '${GENOME_NAME}.fasta' --filterName DepthFilter --filterExpression 'DP < {{filtration.rd}} || AF < {{filtration.af}}'\"" > temp.sh
       sudo unshare -n bash temp.sh
       echo "{\"proc.progress\":0.2,\"proc.rc\":$?}"
-      echo "su $(whoami) -s /bin/bash -c \"java -jar {{ proc.slugs_path }}/GATK-3.2.2/GenomeAnalysisTK.jar -T SelectVariants --variant '${VARIANTS}_filtered.vcf' -o '${VARIANTS}_selected.vcf' -R '${GENOME_NAME}.fasta' -ef\"" > temp.sh
+      echo "su $(whoami) -s /bin/bash -c \"java -jar GenomeAnalysisTK.jar -T SelectVariants --variant '${VARIANTS}_filtered.vcf' -o '${VARIANTS}_selected.vcf' -R '${GENOME_NAME}.fasta' -ef\"" > temp.sh
       sudo unshare -n bash temp.sh
       echo "{\"proc.progress\":0.25,\"proc.rc\":$?}"
 
       # annotation
-      perl "{{ proc.slugs_path }}/SNPdat-1.0.5/SNPdat_v1.0.5.pl" -i "${VARIANTS}_selected.vcf" -g "{{gtf.gtf.file}}" -f "${GENOME_NAME}.fasta" -o "${OUTPUT}" -s "${SUMMARY}"
+      perl "SNPdat_v1.0.5.pl" -i "${VARIANTS}_selected.vcf" -g "{{gtf.gtf.file}}" -f "${GENOME_NAME}.fasta" -o "${OUTPUT}" -s "${SUMMARY}"
```

Comment:
I think we should just fix the she-bang in the [upstream's Perl script](https://snpdat.googlecode.com/files/SNPdat_v1.0.5.pl) and execute `SNPdat_v1.0.5.pl` directly.

I would even go further and rename it to `SNPdat.pl` since we don't reference tools by their version anymore.